ADR/ADR028-use-notify-for-users.md

```diff
@@ -0,0 +1,24 @@
+# ADR029: Use GOV.UK Notify for emails sent to users for simple notifications
+
+Date: 2024-03-28
+
+## Status
+
+Accepted
+
+
+## Context
+
+[Submission emails](https://github.com/alphagov/forms-runner/blob/main/app/mailers/form_submission_mailer.rb) and [submission confirmation emails](https://github.com/alphagov/forms-runner/blob/main/app/mailers/form_submission_confirmation_mailer.rb) in runner and the [OTP to verify a change in submission email via admin](https://github.com/alphagov/forms-admin/blob/main/app/mailers/submission_email_mailer.rb) are sent via GOV.UK Notify. Notifications to users relating to account management (such as when they are upgraded from a trial to full user account), will be sent using GOV.UK Notify.
+
+We currently only use Amazon SES for Auth0 One Time Passcode emails, as we could not integrate GOV.UK Notify with Auth0 as they have a higher TLS requirement.
+
+This is an addendum to [ADR019](ADR019-user-amazon-ses.md).
+
+## Decision
+
+We will use GOV.UK Notify where the service is appropriate and does not require the additional features that Amazon SES offers.
+
+## Consequences
+
+We keep using a free and simpler mechanism for sending emails to our users.
```

Comment:
Just wondering in order to be balanced, whether its worth mentioning some of the following?
- GOV.UK Forms does not need to invest in building additional monitoring, alert and administrative interfaces related to sending email which GOV.UK Notify already provide.
- GOV.UK Forms will be dependent upon the support, maintenance and feature roadmap of GOV.UK Notify, for example if and when they offer enforced TLS to the recipients inbox.